lib/initPTB.m

```diff
@@ -92,7 +55,8 @@
         
         
     else
-        Screen('Preference', 'SkipSyncTests', 0);
+         Screen('Preference', 'SkipSyncTests', 0); 
```

Comment:
My PTB crashes here because it doesn't pass the synctests. 
I expect many people would have similar issue on their computers. 
However, I'm not sure we need such precision for the screen in this experiment. Perhaps we could just skip it in both modes?